packages/report/src/runtime/IReporter.ts

```diff
@@ -67,4 +92,6 @@ export interface IReporter {
 	addListener?(event: string | symbol, listener: (...args: any[]) => void): this
 	on?(event: string | symbol, listener: (...args: any[]) => void): this
 	once?(event: string | symbol, listener: (...args: any[]) => void): this
+	setWorker?(worker: WorkerReport): void
+	sendReport(msg: string, logType: string): void
```

Comment:
If it can only be `action`, `iteration` or `measurement`, then
```suggestion
	sendReport(msg: string, type: 'action' | 'iteration' | 'measurement'): void
```
then there's no need for `ACTION`, `ITERATION` and `MEASUREMENT` above

Reply:
updated